splash/qtrender_lua.py

```diff
@@ -221,6 +248,10 @@ def init_dispatcher(self, return_func):
         """
         self._return = return_func
 
+    @attr_function(attr_name='js_enabled', default=True)
```

Comment:
Yes, I agree. This kind of felt like the wrong way to specify default value to me too, but I couldn't think of a proper alternative.

Would it be better if I set these attributes and then explicitly set the default values in `__init__()`
like this?
``` python   
def __init__(....):
    self.images_enabled = True   # Specify the default value
    ....
    self.set_images_enabled(self.images_enabled)  # Set default value explicitly
```